src/analysis/object.rs

```diff
@@ -66,20 +66,20 @@ pub fn class(env: &Env, obj: &GObject, deps: &[library::TypeId]) -> Option<Info>
 
     let supertypes = supertypes::analyze(env, class_tid, &mut imports);
 
-    let mut has_children = obj.force_trait;
+    let mut generate_trait = obj.generate_trait;
 
     for child_tid in env.class_hierarchy.subtypes(class_tid) {
         let child_name = child_tid.full_name(&env.library);
         let status = env.config.objects.get(&child_name)
             .map(|o| o.status)
             .unwrap_or_default();
         if status.normal() {
-            has_children = true;
+            generate_trait = true;
             break;
         }
     }
```

Comment:
Why? If obj.generate_trait=false (i.e. set false in the configuration), we would still want to generate a trait if it is actually needed. Or not?